AdtSampleApp/SampleClientApp/BuildingScenario.cs

```diff
@@ -1,75 +1,70 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-using Azure;
+﻿using System.Threading.Tasks;
 
 namespace SampleClientApp
 {
     public class BuildingScenario
     {
         private CommandLoop cl;
-        public BuildingScenario(CommandLoop cl_)
+        public BuildingScenario(CommandLoop cl)
```

Comment:
style change: we don't name variables with trailing underscores. If we don't name member variables with an underscore prefix (one common pattern), then we disambiguate with "this.".